src/wh_client_cryptocb.c

```diff
@@ -230,39 +301,91 @@ int wh_Client_CryptoCb(int devId, wc_CryptoInfo* info, void* inCtx)
                 WC_ALGO_TYPE_PK,
                 WH_PACKET_STUB_SIZE + sizeof(packet->pkRsakgReq),
                 (uint8_t*)packet);
+            printf("RSA KeyGen Req sent:size:%u, e:%u, ret:%d\n",
+                    packet->pkRsakgReq.size, packet->pkRsakgReq.e, ret);
```

Comment:
should be guarded by  `DEBUG_CRYPTOCB_VERBOSE`

Reply:
My bad.  Likely to remove all this debug prints.  Should left the PR in draft form. Let's get this one done as is as i'm moving on to ECC.